app/scss/pages/_home.scss

```diff
@@ -0,0 +1,123 @@
+.rooms {
+	&__inner {
+		padding: 60px 0 95px;
+		@media (max-width : 680px) {
+			padding: 45px 0 65px;
+		}
+		@media (max-width : 320px) {
+			padding: 35px 0 45px;
+		}
+	}
+	&__title {
+		margin-bottom: 90px;
+		font-size: 25px;
+		font-weight: 400;
+		text-align: center;
+		@media (max-width : 680px) {
+			margin-bottom: 25px;
+		}
+		@media (max-width : 480px) {
+			font-size: 17px;
+		}
+	}
+}
+.room {
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 70px;
+	position: relative;
+	&::before {
+		content: "";
+		border: 1px solid #14274A;
+		position: absolute;
+		height: 230px;
+		left: -35px;
+		bottom: 103px;
+		@media (max-width : 1170px) {
+			display: none;
+		}
+	}
+	@media (max-width : 990px) {
+		flex-wrap: wrap;
+		justify-content: center;
+	}
+	@media (max-width : 480px) {
+		margin-bottom: 35px;
+	}
+	&__info {
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		margin-right: 70px;
+		@media (max-width : 990px) {
```

Comment:
<img width="804" alt="Screenshot 2021-01-24 at 22 34 56" src="https://user-images.githubusercontent.com/13525860/105642775-8c7b6580-5e94-11eb-9eee-c5441d7aa218.png">

Not really good
Need to make images a bit less and text will be wide everything should look good

Reply:
done